packages/libs/react-ui/src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,43 @@
+import { accordionWrapperClass } from './Accordion.css';
+import { AccordionSection, IAccordionSection } from './AccordionSection';
+
+import React, { FC, useState } from 'react';
+
+export interface IAccordionProps {
+  className?: string;
+  sections: Omit<IAccordionSection, 'isOpen' | 'onToggle'>[];
+  linked?: boolean;
+}
+
+export const Accordion: FC<IAccordionProps> = ({ sections, linked = true }) => {
+  const [expandedSections, setExpandedSections] = useState<number[]>([]);
+  const handleOpen = (index: number): void => {
+    if (linked) setExpandedSections([index]);
+    else setExpandedSections([...expandedSections, index]);
+  };
+  const handleClose = (index: number): void => {
+    setExpandedSections(expandedSections.filter((item) => item !== index));
+  };
+
+  const isOpen = (index: number): boolean => expandedSections.includes(index);
+
+  const handleToggle = (index: number): void => {
+    if (isOpen(index)) handleClose(index);
+    else handleOpen(index);
+  };
+
+  return (
+    <div className={accordionWrapperClass} data-testid="kda-accordion-wrapper">
+      {sections.map((section, index) => (
+        <AccordionSection
+          isOpen={isOpen(index)}
+          onToggle={() => handleToggle(index)}
+          title={section.title}
+          key={index}
```

Comment:
we should avoid using indices as keys. Is there a value in section that can be used for this?

Reply:
well, each element only has a title which is a react node, and nodes can't be used for keys, I can assign id, but then developer will be forced to add id to items of array which has no usage but the key!